BitmarkRegistry/Models/Enums/AssetType.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  AssetType.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/26/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import UIKit
+import MobileCoreServices
+
+enum AssetType {
+  case medical
+  case health
+  case image
+  case video
+  case doc
+  case zip
+  case unknown
+
+  func thumbnailImage() -> UIImage? {
+    switch self {
+    case .medical:
+      return UIImage(named: "medical-record-asset-type")
+    case .health:
+      return UIImage(named: "health-data-asset-type")
+    case .image:
+      return UIImage(named: "image-asset-type")
+    case .video:
+      return UIImage(named: "video-asset-type")
+    case .doc:
+      return UIImage(named: "doc-asset-type")
+    case .zip:
+      return UIImage(named: "zip-asset-type")
+    case .unknown:
+      return UIImage(named: "unknown-asset-type")
+    }
+  }
+
+  static func get(withSource source: String) -> AssetType? {
+    switch source {
+    case "Health Records", "Medical Records":
+      return .medical
+    case "Health Kit", "Health":
+      return .health
+    default:
+      return nil
+    }
+  }
+
+  static func get(withFilePath filepath: String) -> AssetType {
+    let pathExtension: CFString = filepath.pathExtension as CFString
+    guard let uti = UTTypeCreatePreferredIdentifierForTag(kUTTagClassFilenameExtension, pathExtension, nil)?.takeRetainedValue() else { return .unknown }
```

Comment:
Do you think we should support `AssetType` on `AssetR`? I don't know if this would be a heavy task as we do it again and again when loading assets.

Reply:
I'm ok for updating this.
as I tested this is not really heavy task; but I am ok to improve this :)